### ISSUE TITLE
Liks to HTML files are updated even when using dontRenameFile

### ISSUE THREAD

**Issue by icrwly** [smysnk/gulp-rev-all#142]
I'm using dontRenameFile: [/^\/favicon.ico$/g, '.html'] which works, but links to those files are still being updated.  For example:

page.html is still page.html but <a href='page.html'> in index.html is now <a href=page.e0e74961.html'>

**Comment by mice33** [comment]
Same problem

**Comment by toineghi** [comment]
+1

**Comment by 2ec0b4** [comment]
Hello,
I think the parameter `'.html'`is wrong, you should probably use something like this `'/^.+\.html$/g'`

**Comment by 2ec0b4** [comment]
My bad, i didn't see that the readme specify the possibility to use `'.html'`

**Comment by brebaje** [comment]
+1 any workaround?

**Comment by ineghi** [comment]
True, a simple dontUpdateReference: [/^\/favicon.ico$/g, '.html'] works.

**Comment by mix3d** [comment]
shouldn't dontUpdateReference happen automatically for any file that is specified in the dontRenameFile array?
